### PR TITLE
Doc issue/31 - main readme

### PR DESCRIPTION
**Pull request by tjex**
just needing the missing statements of contributions. 
also, if someone could give it a read over and check that the programming side of things makes sense, would be ace.

### CODE REVIEW

**Comment by andersblomqvist** [comment]
Looks good but I would also like to have:

* Build instructions
* Token creation and that it needs to be set in environment variables

File: README.md [hunk 1]
```diff
@@ -1,2 +1,289 @@
...
+1. code is pushed to Github from a local repo
+2. Github registers the push and sends a json package to the server
+3. the server clones the repo, 
+4. builds the resultant code, 
+5. formats a response destined for Github based on the build result and
+6. sends the response (headers and json) back to Github
+
+The response back to Github contains the status of the build, which can either be 
+`error`, `faliure`, `pending` or `success`. This response is displayed 
+alongside the commit in Github.
```
Comment (coffecup25) [inline README.md:30]: We only send FAILURE and SUCCESS actually

**Review by coffecup25 — CHANGES_REQUESTED** [review]
We have to add the following to the Statement of Contributions:

Property: the group is creative and proactive, they have done something remarkable, for which they are proud.

Assessment: the students claim in their "Statement of contributions" that they have done something valuable and remarkable in their project. The grader subjectively assess whether it counts towards a distinction.

File: README.md (hunk 1 above)
Comment (tjex) [reply README.md:30]: yeah, but there I was more explaining what a webhoook can actually do. I guess the language is a bit unclear

Comment (coffecup25) [reply README.md:30]: ahh yeah I get it now, no need to change then

**Review by andersblomqvist — APPROVED** [review]
Nice additions.

**Review by coffecup25 — APPROVED** [review]
Good job!

**Pull request merged**